src/sidebar/store/modules/test/annotations-test.js

```diff
@@ -1,28 +1,16 @@
 'use strict';
 
-var redux = require('redux');
```

Comment:
A while back a `createStore` utility was introduced to create a store from a set of _modules_. It is used by both the actual application and several of the tests. Here I'm changing the tests for the annotations module to use it for consistency.